.github/workflows/unit.yml

```diff
@@ -0,0 +1,25 @@
+name: unit
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - '**'
+      - '!doc/**'
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '~1.16'
+      - name: Install Kubebuilder
+        run: |
+          os=$(go env GOOS)
+          arch=$(go env GOARCH)
+          curl -L https://go.kubebuilder.io/dl/2.3.1/${os}/${arch} | tar -xz -C /tmp/
+          sudo mv /tmp/kubebuilder_2.3.1_${os}_${arch} /usr/local/kubebuilder
```

Comment:
A bit surprised we have a hard kubebuilder dependency to run our unit tests, but I seem to recall that being introduced sometime last year. Not really a huge deal I suppose.

Reply:
I didn't dive too much into why we have that explicit dependency as I was only copying the content in the base.Dockerfile, which is the container image that the unit prow job would be run on. Once this and the o/release PR merges, we can re-evaluate whether we need that build root Dockerfile anymore (along with the root Dockerfile, etc.).